package.json

```diff
@@ -193,20 +193,23 @@
         "mindaro-dev.file-downloader"
     ],
     "devDependencies": {
+        "@types/glob": "^7.1.1",
+        "@types/got": "^9.6.12",
         "@types/mocha": "^10.0.1",
         "@types/node": "^18.11.17",
         "@types/node-fetch": "^2.6.2",
         "@types/vscode": "1.74.0",
+        "@vscode/test-electron": "^2.3.3",
```

Comment:
Replaced deprecated ```vscode-test``` with ```@vscode/test-electron``` and refactored the ```TestRunHost.ts``` to work with current vscode/test-electron package. Also removed ```mocha-trx-reporter``` which was of no use.